index.html

```diff
@@ -0,0 +1,48 @@
+<html>
+    <head>
+        <link rel="stylesheet" href="styles.css">
+        <title>HTML project</title>
+    </head>
+    <body>
+        <main id="main">
+            <h1 id="title">Dr. Norman Borlaug</h1>
+            <p>The man who saved a billion lives</p>
+            <figure id="img-div">
+            <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="picture">
+            <figcaption>Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</figcaption>
+            </figure>
+
+            <h2>Here's a time line of Dr. Borlaug's life:</h2>
+
+            <div id="tribute-info">
```

Comment:
Please wrap this list part into `section` tag instead of `div` tag. This is for semantic reasons and the right position for convention and user experience.